backend/tests/test_rag_system.py

```
@@ -0,0 +1,507 @@
+        mock_db.refresh = AsyncMock()
+        repo = VectorRepository(mock_db)
+
+        vector_data = {
+            "flow_id": 1,
+            "config_id": 1,
+            "embedding": [0.1] * 1536,
+            "reference_input": {"plain_text": "test content", "text_blocks": []},
+            "reference_output": {"invoice_number": "INV-001"},
+        }
+
+        result = await repo.store_training_vector(**vector_data)
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)